### PR TITLE
docs: Update main readme

### PR DESCRIPTION
**Pull request by psomhorst**
Update to the main readme with a short description, updated install instructions and a best practices badge.

Replaces #232.

### CODE REVIEW

**Comment by psomhorst** [comment]
Closes #231

**Review by DaniBodor — APPROVED** [review]
Looking good!

Some minor comments, and I added a commit to organize and fix some of the badges.

File: README.md [hunk 1]
```diff
@@ -1,41 +1,59 @@
...
+
+[eit_dash](https://github.com/EIT-ALIVE/eit_dash) provides an accompanying GUI. 
 
-The project setup is documented in [project_setup.md](project_setup.md). Feel free to remove this document (and/or the link to this document) if you don't need it.
 
 ## Installation
 
-Install eitprocessing as follows:
+It is advised to install eitprocessing in a dedicated virtual environment. See e.g. [Install packages in a virtual environment using pip and venv](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/) or
+[Getting started with conda](https://docs.conda.io/projects/conda/en/stable/user-guide/getting-started.html).
```
[the remark this answers is not in the document]
Comment (psomhorst) [reply README.md]: Maybe. However, I don't want to write out methods for venv, pipenv, pyenv, conda, etc. There are so many solutions nowadays that it's hard to find out what everyone is using and writing guides for every platform.

[the remark this answers is not in the document]
Comment (DaniBodor) [reply README.md]: The reason to make it explicit is for people who don't know about virtual environments to not accidentally write into their base env. For this, it's fine to just give one option (usually conda, but any is fine). Usually, users that have preferences for specific methods will know (or look up) how to adapt the code to fit their preference.
You can mention that other options are also available, but I think even that is not necessary.

**Pull request merged**